### DISCUSSION TITLE
V3 Release Date

### DISCUSSION THREAD

**Discussion by TonyValenti** [SixLabors/ImageSharp#2309] [Q&A]
Just wondering when it might land.

**Comment by joaovsoares** [comment]
Likely to be released on January 30th as quoted in the milestone.

https://github.com/SixLabors/ImageSharp/milestone/10
![image](https://user-images.githubusercontent.com/26191248/210081850-83856a6e-a8ef-45d3-ae0e-26a629a566c2.png)

**Reply by felipealbuquerq** [reply]
This information could be in the Readme, as a link for example.

**Reply by JimBobSquarePants** [reply]
Why? Changing the readme at any point triggers a build/nightly-release. Releases are the designated location for this information. 

I'd like to note that this date is subject to change. In fact, it most likely will.

**Reply by felipealbuquerq** [reply]
You don't need to post it now or tomorrow, but when you actually launch a release, take advantage and share this link that demonstrates the milestones in general: https://github.com/SixLabors/ImageSharp/milestones

**Comment by felipealbuquerq** [comment]
@TonyValenti , I don't know if you've seen it, but you can use the alpha while the release doesn't come out.

https://github.com/SixLabors/ImageSharp#installation
